src/views/TimeOutWrapper/index.tsx

```diff
@@ -118,19 +92,34 @@ const TimeOutWrapper = ({ children }: TimeOutWrapperProps) => {
   // Calculates the user's inactivity to display the modal
   useInterval(
     () => {
-      const tokenExpiresIn = calculcateTokenExpiration();
-      if (
-        !activeSinceLastRenew &&
-        tokenExpiresIn > 0 &&
-        tokenExpiresIn < fiveMinutes
-      ) {
-        setTimeRemainingArr(formatSessionTimeRemaining(tokenExpiresIn));
+      if (lastActiveAt.toMillis() + tenMinutes < DateTime.local().toMillis()) {
+        setTimeRemainingArr(formatSessionTimeRemaining(countdown));
         setIsModalOpen(true);
       }
     },
     authState?.isAuthenticated && !isModalOpen ? oneSecond : null
   );
 
+  // If user is authenticated and has a token, renew it forever five
+  // minutes before expiration.
+  // The user's inactivity will log the user out.
+  useEffect(() => {
```

Comment:
Because `autoRenew` and listening to the `tokenManager.on('expired')` doesn't work reliably anymore, I am renewing the token every nine minutes (token length is 14 minutes - 5 minutes = 9). We could probably make it less frequent, but i went with this as a starting number.

The token will continue to renew until the user is inactive for 15 minutes. They will be signed out.